### ISSUE TITLE
Not compatible with Nextcloud 18.0.7

### ISSUE THREAD

**Issue by fafische** [CarnetApp/CarnetNextcloud#117]
**Describe the bug**
Befor updating to Nextcloud 18.0.7, the updater shows a warning that there is no update for Carnet.

**To Reproduce**
- use Nextcloud 18.0.6
- install Carnet-App
- try updating to Nextcloud 18.0.7

**Expected behavior**
version of Carnet-App is compatible with NC 18.0.7 -> no warning for missing app-update

**Desktop (please complete the following information):**
 - OS: WIn10 1909
 - Browser Chrome
 - Version 84.0.4147.89

**Comment by zocker-160** [comment]
I second this issue, but it is odd, because I am on 18.0.4 and I cannot find carnet in the app list anymore

also the download section on their website is empty:
https://apps.nextcloud.com/apps/carnet

normally you can see the supported version of the app there

**Comment by PhieF** [comment]
Yes I'm investigating that, I cannot upload new versions of the app, there is an error

**Comment by zocker-160** [comment]
This issue looks fixed to me, I could upgrade everything without problems. @PhieF

**Comment by GTP95** [comment]
I suggest closing this issue, as it is superseded by issue #173.
